### PR TITLE
[Seed Users] OT245-77 - Create user seed

### PR DESCRIPTION
**Pull request by CamiloLonM**
[https://alkemy-labs.atlassian.net/browse/OT245-77](url)

**Description**

AS admin user
I WANT to view test data
TO be able to test functionalities on the web.

**Criteria of acceptance:**
Create a seeder to populate the database with Users for testing. 10 standard and 10 regular users. Later, add their data in the README.md to allow other Members to use them.

image

image

### CODE REVIEW

**Review by gabodmc — APPROVED** [review]

**Review by miguelpuente — APPROVED** [review]

File: database/seeders/20220729222239-roles.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+
+ await queryInterface.bulkInsert('Roles', [
+  {
+    name: 'Admin',
+    description: 'Administator',
```
Comment (itsnicolearias) [inline database/seeders/20220729222239-roles.js]: Esta genial, solo 2 correcciones "Administrator"

File: database/seeders/20220729222239-roles.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    createdAt: new Date(),
+    updatedAt: new Date(),
+    },
+  {
+    name: 'Standart',
```
Comment (itsnicolearias) [inline database/seeders/20220729222239-roles.js]: y "Standard"

**Review by itsnicolearias — CHANGES_REQUESTED** [review]

**Review by gabodmc — CHANGES_REQUESTED** [review]
Se me pasaron esas correcciones. Me sumo a lo comentado por Niki

File: database/seeders/20220729222239-roles.js (hunk 2 above)
Comment (CamiloLonM) [reply database/seeders/20220729222239-roles.js]: lista, corrección de nombres

**Comment by CamiloLonM** [comment]
lista, corrección de nombres

**Review by itsnicolearias — APPROVED** [review]

**Review by miguelpuente — APPROVED** [review]
Buen ojo Niki!

**Review by DamiBastos — APPROVED** [review]

**Review by HinojosaAcevedoJ — APPROVED** [review]

**Review by gabodmc — APPROVED** [review]

**Pull request merged**